includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -2159,7 +2170,7 @@ protected function prepare_payment_information_from_request( WC_Order $order ) {
 				$order,
 				$payment_method_details
 			);
-			$payment_information['capture_method']               = $capture_method;
+				$payment_information['capture_method']           = $capture_method;
```

Comment:
I'm seeing that same indenting issue here. @rafaelzaleski let me know if this is an unexpected indent for you as well here?

Reply:
This is the result of running `npm run lint:php-fix`. Could it be that the rules in our linter are not properly configured?

Reply:
I was wondering if that was the case, yeah it seems something might be off. Reviewing the linter rules during our meetup would be a good exercise.